homeassistant/components/climate/ecobee.py

```diff
@@ -145,12 +145,30 @@ def current_temperature(self):
     @property
     def target_temperature_low(self):
         """Return the lower bound temperature we try to reach."""
-        return int(self.thermostat['runtime']['desiredHeat'] / 10)
+        if self.operation_mode == 'auto':
```

Comment:
Hmm.... not sure how that happened... In any case, there were some further updates, plus I changed this using STATE_AUTO.

You are very thorough. Thanks.